task/yq/0.1/yq.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: yq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: yq
+    tekton.dev/displayName: "YQ replace"
+spec:
+  description: >-
+    This task can be used to replace fields in YAML files. For example for altering helm charts on GitOps repos.
+  workspaces:
+    - name: shared-workspace
+      description: A workspace that contains the fetched git repository.
```

Comment:
This desc is git-specific but presumably yq can be used with non-git sources too?

Suggest naming this workspace something like "content" or "source" with a description like "A workspace that contains the file you want to modify."